packages/oss-console/src/components/Breadcrumbs/registry/index.ts

```diff
@@ -106,7 +106,7 @@ export class BreadcrumbRegistry {
 
   static makeUrlSegments(location: Location, projectId = '', domainId = '') {
     const pathName = location.pathname;
-    const basePath = process.env.BASE_PATH || '/console';
```

Comment:
I believe BASE_PATH is a typo here, since the `process.env` object looks like
```
{
    NODE_ENV: "production",
    ADMIN_API: "//localhost:30080",
    ADMIN_API_URL: "localhost:30080",
    BASE_URL: "/console",
    BASE_HREF: "",
    DISABLE_CONSOLE_ROUTE_PREFIX: "",
    MAINTENANCE_MODE: ""
}
```
(in the case of a custom path, it looks like `... BASEURL: "/custom-path/console"`

And double checking the repo code, BASE_PATH has no other instance of usage, pointing to this being a potential typo.